docs/getting_started/faq.rst

```diff
@@ -89,15 +94,12 @@ import with Modin import:
 
 Which execution engine (Ray or Dask) should I use for Modin?
 """"""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-
-Whichever one you want! Modin supports Ray_ and Dask_ execution engines to provide an effortless way
-to speed up your pandas workflows. The best thing is that you don't need to know
-anything about Ray and Dask in order to use Modin and Modin will automatically
-detect which engine you have
-installed and use that for scheduling computation. If you don't have a preference, we recommend
-starting with Modin's default Ray engine. If you want to use a specific
-compute engine, you can set the environment variable ``MODIN_ENGINE`` and
-Modin will do computation with that engine:
+Modin lets you effortlessly speed up your pandas workflows with either Ray_'s or Dask_'s execution engine.
+You don't need to know anything about either engine in order to use it with Modin. If you only have one engine
+installed, Modin will automatically detect which engine you have installed and use that for scheduling computation.
+If you don't have a preference, we recommend starting with Modin's default Ray engine.
+If you want to use a specific compute engine, you can set the environment variable ``MODIN_ENGINE``
```

Comment:
can we point to the configs here instead of environment variables?

```python
from modin.config import Engine

Engine.put("ray")
Engine.put("dask")
```

Reply:
I think we can keep both.

Reply:
Should I also change this in the README so removing both the `export MODIN_ENGINE` and `os.environ` code snippets?